packages/framework/tests/Unit/Component/Grid/GridTest.php

```diff
@@ -11,7 +11,7 @@
 use Symfony\Component\HttpFoundation\Request;
 use Symfony\Component\HttpFoundation\RequestStack;
 use Symfony\Component\Routing\Router;
-use Twig_Environment;
+use Twig\Environment;
```

Comment:
Yeah, but if you look at using `Twig_`, you will find that it is used in `Component\Translation` and when you use `Twig_` objects, it's because of the jms / translation-bundle whitch requires `Twig_` objects in their the methods.

Reply:
Hmm... What about
- packages/framework/src/Component/Grid/GridView.php
- packages/framework/src/Model/Feed/FeedRenderer.php
- project-base/tests/App/Functional/Model/Order/Mail/OrderMailTest.php

and maybe others... can you doublecheck, please?

Reply:
I don't know where I was looking. Sorry a622f7219c9df35350be87685179c5201fb4a4aa